pages/common/nix-build.md

```diff
@@ -6,8 +6,8 @@
 
 - Build a Nix expression:
 
-`nix-build '<nixpkgs>' --attr {{firefox}}`
+`nix-build '{{nixpkgs}}' {{[-A|--attr]}} {{firefox}}`
 
 - Build a sandboxed Nix expression (on non-NixOS):
 
-`nix-build '<nixpkgs>' --attr {{firefox}} --option sandbox true`
+`nix-build '{{nixpkgs}}' {{[-A|--attr]}} {{firefox}} --option sandbox true`
```

Comment:
I assume the angle brackets are not part of the syntax, but were intended to be placeholder indicators

Reply:
Apparently it's not https://github.com/tldr-pages/tldr/pull/9365#issuecomment-1294739117